Formula/d/dda.rb

```diff
@@ -0,0 +1,46 @@
+class Dda < Formula
+  desc "Tool for developing on the Datadog Agent platform"
+  homepage "https://github.com/DataDog/datadog-agent"
+  url "https://github.com/ofek/pyapp/releases/download/v0.28.0/source.tar.gz"
+  version "0.23.1"
```

Comment:
why does the version mismatch?

Reply:
I don't understand what that means. Which one of the versions is the actual version of the tool?

Reply:
The version here `0.23.1` is the actual project version.

Reply:
And where can I find the version of the project?